src/store/actions/user.js

```diff
@@ -1,12 +1,43 @@
 /* eslint-disable linebreak-style */
-import { SET_USER } from './actionTypes';
+import axios from 'axios'
+import { SET_USER, FETCH_USER_DATA_REQUEST, FETCH_USER_DATA_SUCCESS, FETCH_USER_DATA_FAILURE, SIGN_USER_OUT } from './actionTypes';
 import { userUiStartLoading, userUiStopLoading } from './ui';
 
 export const setUser = (user) => ({
   type: SET_USER,
   user
 });
 
+export const fetchUserDataRequest = () => ({ type: FETCH_USER_DATA_REQUEST})
+export const fetchUserDataSuccess = (userData) => ({ type: FETCH_USER_DATA_SUCCESS, payload: userData})
+export const fetchUserDataFailure = (error) => ({ type: FETCH_USER_DATA_FAILURE, payload: error})
+export const fetchUserOnline = (error) => ({type: FETCH_USER_ERROR, payload: error});
+
+// 'https://johnerry.000webhostapp.com/placeholder.php?u='
+// 'https://my-json-server.typicode.com/johnerry/json_place_holder/'
+export const fetchUserData = (name) => async (dispatch) => {
+  try {
+    dispatch(fetchUserDataRequest())
+    axios.get('https://johnerry.000webhostapp.com/placeholder.php?u='+name)
+    .then( response => {
```

Comment:
Codacy found an issue: [There should be no space after this paren.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5714105)